### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by YukunYangNPF** [plotly/dash-alternative-viz#7]
pip3 install dash-alternative-viz
ERROR: Could not find a version that satisfies the requirement dash-alternative-viz (from versions: none)
ERROR: No matching distribution found for dash-alternative-viz

environment: Python 3.8.6

**Comment by vishaltambe70** [comment]
That's right, this project is a proof-of-concept that we haven't published to PyPI. You can certainly use it either by cloning the repo and running pip install -e . or install directly from github:
pip install git+https://github.com/plotly/dash-alternative-viz.git#egg=dash_alternative_viz
